### PR TITLE
changes to model_manager to support optional rel prop

### PR DESCRIPTION
**Pull request by PranjaliParnerkar**
Hi @WillMcDGSK Could you please review.
@paltusplintus FYI.

This PR is for this [PBI](https://dev.azure.com/DevOps-RD/Clinical%20Linked%20Data%20Experiments/_sprints/taskboard/Clinical%20Linked%20Data%20Experiments%20Team/Clinical%20Linked%20Data%20Experiments/Sprint%2010.4?workitem=999743).

Changes to model_manager.py to support optional relationship property in cld-vscode-extension.

### CODE REVIEW

File: model_managers/model_manager.py [hunk 1]
```diff
@@ -220,9 +220,9 @@ def create_relationship(self, rel_list: List[List[str]], identifier='label', mat
         """
         Create relationship nodes between two specified classes as defined in rel_list.
         For example:
-            With rel_list = [ ['class1', 'class2', 'example'] ]
+            With rel_list = [ ['class1', 'class2', 'example', 'false'] ]
```
Comment (paltusplintus) [inline model_managers/model_manager.py:223]: @PranjaliParnerkar 
On the tab2neo level I would not like to mandate the optional status of a relationship to be provided.
Could you please implement the function to support both:
- a list of length 3 (without the optional status)
- and a list of length 4 with optional status
?
Would be useful to have unit tests for both scenarios.
Thanks.

Comment (WillMcDGSK) [reply model_managers/model_manager.py:223]: @PranjaliParnerkar To add to this, some current unit tests are failing with this change as they do not expect an optional key in the relationships dicts.

Comment (PranjaliParnerkar) [reply model_managers/model_manager.py]: I have made changes to create_relationship() to allow list with and without optional rel and have updated the test cases as well. Please have a look and let me know if changes are required. Thanks!

**Review by WillMcDGSK — APPROVED** [review]
Looks good!

**Pull request merged**